package/yast2-nfs-client.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Mon May 11 15:04:25 UTC 2020 - Stefan Hundhammer <shundhammer@suse.com>
+
+- Fixed crash when using [Cancel] in the "Add" dialog (bsc#1170447)
```

Comment:
Just for the record, it fixed also https://bugzilla.suse.com/show_bug.cgi?id=1166285 which is essentially the same but reported a month before.